test/stdlib/ErrorBridged.swift

```diff
@@ -767,4 +767,29 @@ ErrorBridgingTests.test("@objc error domains for nested types") {
               String(reflecting: NonPrintAsObjCError.self))
 }
 
+@inline(never)
+@_optimize(none)
+func anyToAny<T, U>(_ a: T, _ : U.Type) -> U {
```

Comment:
This doesn't seem to be used. Is it vestigial or is there supposed to be more testing that uses it?

Reply:
Its vestigial. Can you tell I did this late at night? 🤦‍♂

Reply:
If that's the worst thing that happened then I'd say you're doing pretty well.

Reply:
https://github.com/apple/swift/pull/28357